opendatahub-io/data-science-pipelines-operator
# Update integration tests to use TLS enabled DB/storage

## The issue resolved by this Pull Request:
Resolves https://issues.redhat.com/browse/RHOAIENG-4254

## Description of your changes:


- Generated certs for Minio and MariaDB external. Stored these certs in secrets and configmaps, and added them to respective deployments as volumes and volume mounts, so that both Minio and MariaDB are TLS enabled.
- Stored the signing certificate in a configmap and applied that in the namespace where DSPA with external connections is being deployed.

## Testing instructions

Follow steps in the GH workflow and accordingly deploy and set up changes from this PR on your cluster. Make sure all DSPA resources come up correctly, check DSPO pod logs to ensure connection to TLS enabled Minio and MariaDB is established.

## Checklist
